tests/unit_tests/conftest.py

```diff
@@ -746,15 +759,16 @@ def check_mysql_health(num_retries: int = 1, sleep_time: int = 1):
 def check_server_up(
     base_url: str,
     wandb_server_tag: str = "master",
+    wandb_server_pull: str = "missing",
```

Comment:
Actually, can you please make it a `Literal["missing", "always"]`?

Reply:
Technically there are 3 categories I can add all of these